Article: "Many to Many Relationship with ABP and EF Core"

EngincanV opened the pull request:
Resolves #9975

EngincanV committed:
Create post for "Many-to-many relationship" article

EngincanV committed:
Update the many-to-many relationship article

EngincanV committed:
Update the "many-to-many relationship" article

EngincanV committed:
Finalize the "many-to-many relationship" article

EngincanV committed:
Update POST.md

EngincanV committed:
Update POST.md

enisn commented on docs/en/Community-Articles/2021-10-31-Many-to-Many-Relationship-with-ABP-and-EF-Core/POST.md:
```diff
@@ -0,0 +1,1572 @@
+
+#### Model Classes and Mapping Configurations
+
+Create a folder named **Modals** and add a class named `CategoryViewModel` inside of it. We will use this view modal class to determine which categories are selected or not in our Create/Edit modals.
```
I think it is `**Models**`

```suggestion
Create a folder named **Models** and add a class named `CategoryViewModel` inside of it. We will use this view modal class to determine which categories are selected or not in our Create/Edit modals.
```

enisn commented on docs/en/Community-Articles/2021-10-31-Many-to-Many-Relationship-with-ABP-and-EF-Core/POST.md:
```diff
@@ -0,0 +1,1572 @@
+
+Create a razor page named **CreateModal.cshtml** (and **CreateModal.cshtml.cs**).
+
+* **CreateModal.cs**
```
```suggestion
* **CreateModal.cshtml**
```

braim23 committed:
test

enisn reviewed (changes requested):
When I run the project at the first time. I couldn't see any category while creating a book. Also, the sample doesn't include a category creation page. So, what can I do? Should I create some categories into SQL manually or we should add something more into the example:

- A minimal data-seeder can be added to create some sample categories.
or
- A category creation page can be added into the example

EngincanV commented:
@enisn I did not include a category page in the article on purpose. It would be longer if I included the category page. But it is a good idea to seed initial data for category and author, I will seed some initial data for them on my repo and also update the article.

enisn commented:
Yeah, a minimal DataSeedContributor would be better to see how it works on the first run 🙂

braim23 committed:
Update POST.md

The article didn't have major problems, just very tiny details and a typo or two.

braim23 committed:
final update

just removing the dot from the test

EngincanV committed:
Update POST.md

EngincanV committed:
Merge pull request #10621 from abpframework/article-change

Hamza's edit

EngincanV committed:
Update POST.md
